src/plugins/ui_stateful/sheetview.ts

```diff
@@ -252,9 +252,6 @@ export class SheetViewPlugin extends UIPlugin {
       case "UNFREEZE_COLUMNS_ROWS":
         this.resetViewports(this.getters.getActiveSheetId());
         break;
-      case "DELETE_SHEET":
-        this.sheetsWithDirtyViewports.delete(cmd.sheetId);
-        break;
```

Comment:
why ?

Reply:
As the commit message says, the switch contained the `case "DELETE_SHEET"` twice. The second was thus never used.